### PR TITLE
Adding a new content page for women in the readme.md

### PR DESCRIPTION
**Pull request by marqueslu**
I found a page that i think thats is an interesting page for the women in the computation area, and i added this in the readme.md

### CODE REVIEW

**Comment by harsh183** [comment]
Looks good. @cristianoliveira ?

File: README.md [hunk 1]
```diff
@@ -291,6 +291,9 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     The Womoz Blog. Program updates and news from the women and Mozilla project.
 
+  - [InspirAda na Computação](https://inspiradanacomputacao.com/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+  - The InspirAda na Computação is a blog made by Lidiane Monteiro, a PyLady. The blog is focused on the space of the women in the computation.
```
Comment (cristianoliveira) [inline README.md]: Please remove the "-" at the beginning of the description

**Comment by cristianoliveira** [comment]
Goood! Merging!

File: README.md [hunk 2]
```diff
@@ -292,7 +292,8 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
     The Womoz Blog. Program updates and news from the women and Mozilla project.
 
   - [InspirAda na Computação](https://inspiradanacomputacao.com/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
-  - The InspirAda na Computação is a blog made by Lidiane Monteiro, a PyLady. The blog is focused on the space of the women in the computation.
+   
+    The InspirAda na Computação is a blog made by Lidiane Monteiro, a PyLady. The blog is focused on the space of the women in the computation.
```
Comment (cristianoliveira) [inline README.md]: From original: Compartilhamos conteúdo como forma de estimular mulheres para área de computação, promover o acesso ao conhecimento técnico e trazer como resultado a igualdade de oportunidades

How about:
The blog aims to share content in order to inspire women to become interested in the tech area. It promotes the access to the tech knowledge and brings, as a result, the equality of opportunities.

Comment (harsh183) [reply README.md]: Yes, that looks much better actually.

**Comment by marqueslu** [comment]
ok, i will go apply this changes

**Comment by cristianoliveira** [comment]
Thanks for your contributions!

**Comment by marqueslu** [comment]
was a pleasure to contribute to this project

**Pull request merged**